### DISCUSSION TITLE
Using the Illumina BaseSpace Sequence Hub SPAdes Genome Assembler v3.9.0

### DISCUSSION THREAD

**Discussion by kdorval** [ablab/spades#819] [Q&A]
I have MiSeq data from a 2x301 from bacterial shot-gun libraries that I ran through the Spades Genome Assembler app in Illumina BaseSpace. I'm looking for a single contig/scaffold FASTA as the output from the SPADES app but only obtained a contigs.fasta and scaffolds.fasta files which have contig sequences  (ie. not continuous sequence). Can anyone help me with this?

**Comment by asl** [comment]
The result of the assembly is the set of contigs. What else you're looking for?

**Comment by kdorval** [comment]
Thanks Anton! How can I tell how large the de novo genome is?

Best,
Kim
Sent from my iPhone
Kim Dorval, PhD
Staff Clinical FAS
647.622.8897
________________________________

**Reply by asl** [reply]
There is no need for this. SPAdes does not require such information anywhere.

**Comment by kdorval** [comment]
I’m looking for Spades to tell me the size in bp of the de novo genome it assembled. I’m not looking to input a genome size into Spades. How do people understand the size of the assembled de novo genome? The contig file has rows and rows of sequence separated by breaks, shouldn’t the software provide the combined total length of these sequence rows?


Best,
Kim

Kim (Fazil) Dorval, PhD
647.622.8897
***@***.***

**Reply by asl** [reply]
There are many tools that can show this for you. E.g. QUAST. Note that the length of the assembly != the length of the genome

**Comment by kdorval** [comment]
Thank you Anton, that’s what I was after.


Best,
Kim

Kim (Fazil) Dorval, PhD
647.622.8897
***@***.***
